CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ Please add to the relevant subsections under Unreleased below on every PR where
 
 ## Unreleased
 
+## [0.17.7] - 2024-08-15
```

Comment:
I don’t think this is correct if you do RC

Reply:
sure, but this is for the actual release, i kicked it off in parallel: https://github.com/wandb/wandb/actions/runs/10410878036